coordinator/api/serializers/study.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from coordinator.api.models import Study
+
+
+class StudySerializer(serializers.HyperlinkedModelSerializer):
+
+    version = serializers.CharField(source='latest_version', allow_blank=True)
+
+    class Meta:
+        model = Study
+        fields = ('kf_id', 'name', 'version', 'visible',
+                  'deleted', 'created_at')
```

Comment:
Don't we also need `read_only_fields = ('kf_id', 'created_at')`?

Reply:
The study uses a ReadOnly view so everything is read only.